portfolio/src/main/webapp/repeatedElements.js

```diff
@@ -12,83 +12,134 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+// TODO: rewrite header/footer to use insertBefore and insertAfter
+// TODO: so I don't have to call footer and header in each new page...
+// TODO: will come back to later; a little too much rn
+
+
+// FOOTER AND HEADER CONSTANTS
+const LINKEDINDEST = "https://linkedin.com/in/robert-marcus-8b443b192";
+const LINKEDINIMGSRC = "images/linkedin_logo.png";
+
+const GITHUBDEST = "https://github.com/rob-marcus";
+const GITHUBIMGSRC = "images/github_logo.png";
+
+const LOGOSLIST = {"linkedin": {"dest": LINKEDINDEST, "src": LINKEDINIMGSRC},
```

Comment:
the values here don't need to be constants since they are only used here.  You can directly write them here.